src/store/modules/purchaseOrder/mutation-types.ts

```diff
@@ -1,3 +1,4 @@
 export const SN_PURCHASE_ORDER = 'purchase_order'
 export const PURCHASE_ORDER_UPDATED = SN_PURCHASE_ORDER + '/UPDATED'
-export const PURCHASE_ORDER_DETAIL_UPDATED = SN_PURCHASE_ORDER+ '/DETAIL_UPDATED'
\ No newline at end of file
+export const PURCHASE_ORDER_DETAIL_UPDATED = SN_PURCHASE_ORDER+ '/DETAIL_UPDATED'
+export const UPDATE_PO_PRODUCT_COUNT = SN_PURCHASE_ORDER + '/COUNT_UPDATED'
```

Comment:
Improve mutation type name to have consistency among names.

Reply:
Sir, I have changed the variable name as:- PURCHASE_ORDER_PRODUCT_COUNT_UPDATED